app/controllers/api/v1/bots_controller.rb

```diff
@@ -90,17 +90,17 @@ def create
 
 
   ### TOGGLE user status
-  def opt_route( @user )
-    if @user.active === true
-      @user.active = false
-      if @user.save
+  def opt( user )
+    if user.active === true
```

Comment:
With rails, you can just call `user.active?`. Since `active` is a boolean database column, you can simple add the `?` to have it return true/false.